packages/application-shell/src/components/navbar/use-navbar-state-manager.ts

```diff
@@ -34,10 +34,33 @@ type Action =
   | { type: 'setIsMenuOpenAndMakeExpanderVisible'; payload: boolean }
   | { type: 'reset' };
 
+const cachedIsForcedMenuOpen = window.localStorage.getItem(
+  STORAGE_KEYS.IS_FORCED_MENU_OPEN
+);
+
+const getShouldInitiallyOpen = (): boolean => {
+  if (!isNil(cachedIsForcedMenuOpen)) {
+    return (
+      typeof cachedIsForcedMenuOpen === 'string' &&
```

Comment:
We could use the `JSON.parse` trick here to shorten this if we'd want.

<img width="189" alt="CleanShot 2020-04-09 at 13 14 20@2x" src="https://user-images.githubusercontent.com/1877073/78889503-0c4e5580-7a64-11ea-8b9f-8354bc4b1973.png">

Reply:
https://github.com/commercetools/merchant-center-application-kit/pull/1433/commits/ef314ecc6fb4388e349479cb3c425faac500d649